app/src/androidTest/java/org/mozilla/fenix/syncintegration/SyncIntegrationTest.kt

```diff
@@ -82,6 +84,40 @@ class SyncIntegrationTest {
         }
     }
 
+    @Test
+    fun checkLoginsFromDesktopTest() {
+        homeScreen {
+        }.openThreeDotMenu {
+        }.openSettings {
+            // Necessary to scroll a little bit for all screen sizes
+            TestHelper.scrollToElementByText("Logins and passwords")
+        }.openLoginsAndPasswordSubMenu {
+        }.openSyncLogins {
+            // Tap to sign in from Logins menu
+            tapOnUseEmailToSignIn()
+            typeEmail()
+            tapOnContinueButton()
+            typePassword()
+            tapOnSignIn()
+        }
+        // Automatically goes back to Logins and passwords view
+        settingsSubMenuLoginsAndPassword {
+            mDevice.waitNotNull(Until.findObjects(By.text("Sync logins")), TestAssetHelper.waitingTime)
+            verifyDefaultView()
+            mDevice.waitNotNull(Until.findObjects(By.text("Off")), TestAssetHelper.waitingTime)
```

Comment:
Can both of these object checks for 'sync logins' and 'Off' be moved into the `verifyDefaultView` method? 

I'm not sure what the method does but it might fit better in there, or in their own methods to maintain readability of this test file.

Reply:
Sure, moved there. The `verifyDefaultView` checks the UI, that the options are correct and so